packages/dicomImageLoader/src/imageLoader/internal/xhrRequest.ts

```diff
@@ -148,21 +149,6 @@ function xhrRequest(
         if (options.onprogress) {
           options.onprogress(oProgress, params);
         }
```

Comment:
You said this is not used, but I see it was used here, and we are removing it

Reply:
Not sure what you are askin g- the onprogress is present, was there something else that got removed?

Reply:
If you mean the image load progress, no, we are not using it.  We use the onprogress callback, or a direct listener depending on the handler.